src/assets/scss/app.scss

```diff
@@ -62,3 +62,105 @@
 // Motion UI
 @include motion-ui-transitions;
 @include motion-ui-animations;
+
+$message-new: $white;
+$message: $body-background;
+
+.footer{
+    text-align: center;
+}
+
+.content{
+    width: 80%;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+    white-space: nowrap;
+}
+
+.event-container{
+    h4{
+        color: $custom-menu-color;
+        padding-left: 1rem;
+    }
+
+    .event-message{
+        padding:1rem;
+
+        &:hover{
+            -webkit-box-shadow: 0px 0px 10px 1px rgba(0,0,0,0.5);
+            -moz-box-shadow: 0px 0px 10px 1px rgba(0,0,0,0.5);
+            box-shadow: 0px 0px 10px 1px rgba(0,0,0,0.5);
+            
+            margin-bottom: 5px;
+            /*bottom shadow behind elements ^ fast fix?*/
+        }
+
+        .image{
+                vertical-align: top;
+                width: 32px;
+                height: 32px;
+                .small{
+                    width: 20px;
+                    height: 20px;
+                }
+        }
+
+        &.new{
+            background-color: $message-new;
+        }
+
+        .info{
+             display: inline-block;
+             padding-left:0.5rem;
+
+            .message{
+                margin: inherit;
+                max-width: 550px;
+                width: auto;
+                border: 1px;
+                overflow: hidden;
+                text-overflow: ellipsis;
+            }
+
+            .subject{
+                font-size: 0.7rem;
+                font-weight: bold;
+            }
+        }
+        
+    }
+
+}
+
+@media only screen and (max-width: 800px) {
```

Comment:
"Desktop First" přístup není chtěný, v požadavcích jsem uvedl, že chci začít od mobilu a proč? No velice obecně je to nejlepší postup, jak se v tom neuvařit - s tímhle přístupem vykuchtíš mnohem víc kódu a zcela zbytečně,... Prosím mrkni se na nějaký web, který se touhle problematikou zabývá třeba:
https://zellwk.com/blog/how-to-write-mobile-first-css/